functions/ts/package.json

```diff
@@ -30,6 +30,7 @@
     "@types/jasmine": "^3.5.11",
     "@types/js-yaml": "^3.12.1",
     "@types/node": "^14.0.23",
+    "@types/rw": "file:@types/rw",
```

Comment:
rw should be part of `dependencies` right?

  "dependencies": {
    "rw": "^1.3.3"
  },

Reply:
correct. without that it wasn't possible to use this module.

Reply:
Clarify: `dependencies` section above currently doesn't declare `rw`. How's this working?

Reply:
sorry, just misunderstood the question, thank you for clarification 👍 
You're right - I missed that.
I've corrected and added:
"dependencies": {
...
"rw": "^1.3.3"
...
},

I think that it is working without that, because it has  `"kpt-functions": "^0.14.5"` in `dependencies` and
`kpt-functions` depends on rw: see [here](https://github.com/GoogleContainerTools/kpt-functions-sdk/blob/master/ts/kpt-functions/package.json#L34)